test/Coordinator.js

```diff
@@ -84,6 +84,29 @@ describe("Coordinator Test suit", function () {
                         coordinator.borrowUnderNFT(nftIdFirstPosition, ethers.utils.parseEther("200")),
                     ).to.be.revertedWith("ERC20: transfer amount exceeds balance");
                 });
+
+            describe("Repay lvUSD for position", function () {
+                const lvUSDAmountToRepayInTwoParts = ethers.utils.parseEther("1");
+                before(async function () {
+                    // method under test
+                    await coordinator.repayUnderNFT(nftIdFirstPosition, lvUSDAmountToRepayInTwoParts);
+                });
+                it("Should transfer lvUSD to coordinator address", async function () {
+                    expect(await r.lvUSD.balanceOf(coordinator.address)).to.equal(ethers.utils.parseEther("99"));
```

Comment:
Maybe a quick comment around where `99` is coming from

Reply:
will add